samples/order-processing-client-boxed/src/logger.ts

```diff
@@ -0,0 +1,13 @@
+import pino from "pino";
+
+export const logger = pino({
+  level: process.env["LOG_LEVEL"] || "info",
```

Comment:
```suggestion
  level: process.env["LOG_LEVEL"] ?? "info",
```

process.env should alway be validated using a zod schema

```
const env = EnvSchema.parse(process.env)
```